lib/usd/ui/importDialog/TreeItem.h

```diff
@@ -39,12 +39,13 @@ class MAYAUSD_UI_PUBLIC TreeItem : public QStandardItem
 public:
     using ParentClass = QStandardItem;
 
-    enum class Type
+    enum Column
     {
-        kLoad,
-        kName,
-        kType,
-        kVariants
+        kColumnLoad,
+        kColumnName,
+        kColumnType,
+        kColumnVariants,
+        kColumnLast
     };
```

Comment:
I change this because TreeItem and TreeModel had both an enumwith the same values which were magically assumed to be in the same order. Now there is a single enum, and that enum must be convertible to an int to pass to Qt.